core/subsurfacestartup.c

```diff
@@ -12,9 +12,6 @@
 struct preferences prefs, git_prefs;
 struct preferences default_prefs = {
 	.cloud_base_url = "https://cloud.subsurface-divelog.org/",
-#if defined(SUBSURFACE_MOBILE)
-	.git_local_only = true,
```

Comment:
just a comment, default is as I expected, which makes the earlier comment =false for mobile even more correct.